src/plugins/query_enhancements/server/types.ts

```diff
@@ -3,11 +3,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { PluginSetup } from 'src/plugins/data/server';
+import { ISearchStrategy, PluginSetup } from 'src/plugins/data/server';
 import { DataSourcePluginSetup } from 'src/plugins/data_source/server';
+import { IDataFrameResponse, IOpenSearchDashboardsSearchRequest } from '../../data/common';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface QueryEnhancementsPluginSetup {}
+export interface QueryEnhancementsPluginSetup {
```

Comment:
thanks for updating the interface and deleting the linter disabler! i think we have a couple of these throughout the project that are left there for no reason